tests/Validation/ValidationValidatorTest.php

```diff
@@ -1428,6 +1428,15 @@ public function testValidateEachWithNonArrayWithoutArrayRule()
 		$this->assertFalse($v->passes());
 	}
 
+	public function testValidateEach()
```

Comment:
Please fix the cs. Test methods must be separated by exactly 2 blank lines, not 1.